.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 # Objective
 
-- Describe the objective or issue this PR addresses.
-- If you're fixing a specific issue, say "Fixes #X".
+Why are you making these changes?
+Link any associated issues with "Fixes #X" so they can be automatically closed.
+If this is a particularly complex PR, please add a clear list of the changes made.
 
-## Solution
+Discuss any background information that may be needed for other reviewers to understand the changes you've made.
```

Comment:
“Discuss” feels wrong here ? In my opinion the role of the PR author is not to discuss (yet), but to give enough information to start that discussion.

I suggest using “Detail” or “Add” ?

Reply:
```suggestion
Provide any background information that may be needed for other reviewers to understand the changes you've made.
```